salvo/install_deps.sh

```diff
@@ -14,10 +14,13 @@ fi
 
 /usr/bin/apt update
 /usr/bin/apt -y install \
+  containerd \
```

Comment:
Oooh, not so sure about this one.  Why are we adding this?

Reply:
When I created a new ubuntu instance and ran this ./install_deps.sh it complained before installing apt package `docker.io`. Found some answer on stackover flow which suggested to install containerd before docker.io. For that reason I included containerd as one of the deps package.

Reply:
Keep in mind that this script runs in the CI for Salvo.  We need to make sure that whatever we change here works in the CI when we push to the envoy-perf repository.

In the other PR I suggested to try to detect whether this is running on a standalone host vs CI.  That will influence what packages we install.  For the desktop case we are free to install whatever we want.  For the CI case this list should be as concise as possible so that we don't spend a significant amount of time pulling packages every time we kick off the CI workflows.